bundler/helpers/lib/shared_native_helpers.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require "excon"
+
+module SharedNativeHelpers
+  # Duplicated in lib/dependabot/bundler/file_updater/lockfile_updater.rb
+  # TODO: Stop sanitizing the lockfile once we have bundler 2 installed
+  LOCKFILE_ENDING = /(?<ending>\s*(?:RUBY VERSION|BUNDLED WITH).*)/m.freeze
```

Comment:
Only used in tests, could move to native test helpers

Reply:
Removed ✂️